app/models/katello/candlepin/repository_mapper.rb

```diff
@@ -139,7 +139,7 @@ def kickstart?
       end
 
       def download_policy
-        if katello_content_type == Repository::YUM_TYPE
+        if ::Katello::RootRepository::CONTENT_ATTRIBUTE_RESTRICTIONS[:download_policy].include?(katello_content_type)
           Setting[:default_redhat_download_policy]
         else
           ""
```

Comment:
(Non-blocking): just making sure `""` is still valid for all cases where we'd end up here, right?

Reply:
Our default is to set the download policy to an empty string for repos that don't support download policies, so it should be correct :+1:

Reply:
Empty string should not get sent to pulp anymore since we don't set the download policy for repo types that don't support it.